runtime/src/weights/pallet_teerex.rs

```diff
@@ -52,15 +51,15 @@ impl<T: frame_system::Config> pallet_teerex::WeightInfo for WeightInfo<T> {
 	// Storage: Teerex EnclaveIndex (r:1 w:0)
 	// Storage: Teerex WorkerForShard (r:0 w:1)
 	// Storage: Teerex LatestIpfsHash (r:0 w:1)
-	fn confirm_call() -> Weight {
+	fn confirm_processed_parentchainblock() -> Weight {
```

Comment:
Can't upgrade to the most recent pallet commit because of metadata v14 update. But that should work for the time being..